docs/source/user-guide/configs.md

```diff
@@ -19,46 +19,74 @@
 
 # Configuration
 
-## BallistaContext Configuration Settings
+## Ballista Configuration Settings
 
-Ballista has a number of configuration settings that can be specified when creating a BallistaContext.
+Configuring Ballista is like configuring DataFusion. Apart for a few specific Ballista specific configurations all others are the same like DataFusion.
```

Comment:
I think it needs to be "Apart *from* a few..." and remove the "specific in front of Ballista. So:

Configuring Ballista is similar to configuring DataFusion. Apart from a few Ballista specific configurations, all other options are the same as DataFusion.